Lib/ntpath.py

```diff
@@ -311,6 +311,30 @@ def ismount(path):
         return False
 
 
+# Is the path reserved?
+# This private function is used by pathlib.PureWindowsPath.is_reserved().
+# See GH-88569 for a proposal to make this function public.
```

Comment:
I think we should keep this in `pathlib` until/unless it is decided to take it public. Otherwise having pathlib leak into `os.path` just for purity of an internal API seems unnecessary.